tests/Integration/ApiPlatform/GetHookStatusTest.php

```diff
@@ -38,6 +38,12 @@ public static function setUpBeforeClass(): void
         DatabaseDump::restoreTables(['hook']);
```

Comment:
I think this one cannot work because you didn't create the API Access with the proper hooks, although if you include my suggestion in the ApiTestCase you won't need to because it would be created dynamically when you call `getBearerToken`